devtools/convert_javadoc.html

```diff
@@ -234,23 +240,13 @@
             funcName = "__init__";
             retType = "None";
         }
-        
-        switch (getOutputMethodType()) {
-          case "normal":
-            t = "def " + funcName + "(self" + args +
-                  ") -> " + retType + ":\n        " + t;
-            break;
-          case "classmethod":
-            t = "@classmethod\n    def " + funcName + "(cls" + args +
-                  ") -> " + retType + ":\n        " + t;
-            break;
-          case "staticmethod":
-            if (args.startsWith(",")) {
-              args = args.substring(1);
-            }
-            t = "@staticmethod\n    def " + funcName + "(" + args +
-                  ") -> " + retType + ":\n        " + t;
-            break;
+
+        if (modifiers.includes("static")) {
```

Comment:
Why did you get rid of staticmethod?

Reply:
Because (Guido admitted) staticmethod should never have existed in Python in the first place.